frontend/public/js/app/App.js

```diff
@@ -253,8 +253,11 @@ var App = Marionette.Application.extend(
          */
         finalizeAndQuit: function ()
         {
+            // Assume default parameters for grouping before finalizing 
+            var userSelections = {'distance': '4', 'parts': '4', 'criterion': 'min', 'func': 'Bounding Box', 'graph': '16'}
```

Comment:
Shouldn't these values be asked before finalization? Or, at least, use the default values used in "Group glyphs and reclassify."